### DISCUSSION TITLE
Separate header image for desktop and mobile?

### DISCUSSION THREAD

**Discussion by paulmelis** [pretalx/pretalx#1791] [Q&A] [answered]
We're currently using a wide banner as header image, which works well in a desktop browser:

![image](https://github.com/pretalx/pretalx/assets/1408287/511c3654-b521-4f25-9aff-0a984ff03e94)

However, on mobile this looks like crap, as part of the image overlaps with the event title:

![image](https://github.com/pretalx/pretalx/assets/1408287/c2d02240-1826-4212-a02f-064f0c3f52c7)

I don't immediately see a way in the settings to have separate images. But I do wonder when/how the Event logo is used (which we currently aren't using) and if that could help here? Otherwise, is there a CSS trick that I could use to apply to leave out the header image on mobile?

**Comment by rixx** [comment]
The event title won't be displayed if you upload a logo, so that would probably be a good first step!

I'm afraid different header images won't be a thing any time soon; for the simple reason of "mobile" being ill-defined. We could offer a different header image for, say, devices smaller than 420px width; but then people would ask for 700px or 520px or 320px – I don't think there's a good way to make everybody happy.

If we allow media queries in our custom CSS (sorry, I don't remember off the top of my head), something like this should work:

```css
@media (max-width: XXXpx) {
  #top-bg { background-image: none }
}
```

**Comment by paulmelis** [comment, marked as answer]
Managed to make it work with

```
@media (max-width: 800px) {
  #header-image { width: 0 !important; }
}
```

Thanks!
